package.json

```diff
@@ -15,6 +16,10 @@
   "engines": {
     "node": ">= 12"
   },
+  "exports": {
+    "import": "./mjs/twilio.js",
+    "require": "./es5/twilio.js"
```

Comment:
Which one is picked up in a typescript project?

Reply:
The `main` property (package.json line 5) points to es5. How does it work in conjunction with this section?

Reply:
ESM aware bundlers will pick the `./mjs` folder. W.R.T typescript, I think it will pick up from `./mjs` folder if the target specified in `tsconfig.json` is `module`. We need to confirm this though.

Reply:
> The `main` property (package.json line 5) points to es5. How does it work in conjunction with this section?

`main` is overridden by the above lines in ESM-aware bundlers.